### DISCUSSION TITLE
pigpio log s6-overlay-suexec: fatal: can only run as pid 1

### DISCUSSION THREAD

**Discussion by Rad3d21** [Poeschl/Hassio-Addons#333] [Q&A]
apparently it could be fixed by adding init:false to config.yaml ?
https://developers.home-assistant.io/blog/2022/05/12/s6-overlay-base-images/

Can anyone tell me were this can be fixed/done ?

running supervisor 2022.8.5 on RPI4 
Home Assistant 2022.8.5
Supervisor 2022.08.3
Operating System 8.2
Frontend 20220802.0 - latest

**Comment by Poeschl** [comment]
It must be done in the addon config in this repository. I need to fix this for the mpd addon as well.

The file to change is https://github.com/Poeschl/Hassio-Addons/blob/main/pigpio/config.json

Will fix it the next days.
